xarray/backends/api.py

```diff
@@ -247,7 +247,7 @@ def maybe_decode_store(store, lock=False):
             token = tokenize(file_arg, group, decode_cf, mask_and_scale,
                              decode_times, concat_characters, decode_coords,
                              engine, chunks, drop_variables)
-            name_prefix = '%s:%s/' % (filename_or_obj, group or '')
+            name_prefix = 'open_dataset-%s:%s/' % (filename_or_obj, group or '')
```

Comment:
That would follow current conventions, yes.

    from dask.base import tokenize
    name = 'open-dataset-' + tokenize(filename)